extensions/amp-analytics/0.1/variables.js

```diff
@@ -144,15 +145,36 @@ export class VariableService {
     this.register_('$IF',
         (value, thenValue, elseValue) => value ? thenValue : elseValue);
     this.register_('$REPLACE', replaceMacro);
+
+    /**
+     * Maximum time (since epoch) to report resource timing metrics.
+     * We stop reporting after 1 minute.
+     * @private @const {number}
+     */
+    this.maxResourceTimingReportingTime_ = Date.now() + (60 * 1000);
```

Comment:
This doesn't work. It's each analytics instance has single reportingTime. the `getMacroBindings` only gets called before sending request so we can't start the timer with the `getMacroBindings` as well.

Reply:
good point. moved to requests.js to ensure start time is aligned with the start of amp-analytics element